### PR TITLE
RoR Project: Micro-Redddit

### PR DESCRIPTION
**Pull request by crgc**
### Active records
 For this project, we created Migrations files and Models to Micro-Reddit assignment.

### Migration files created

- create_user
- create_post
- create_comment
- add_user_ref_to_post
- add_user_post_ref_comment

### Model created with validations:

- user
- post
- comment

### CODE REVIEW

**Review by LamiaSristy — COMMENTED** [review]
# Status: Invalid Review Request ✖️

Hello @chuckbuckethead  and @EroAuditore. I can see a lot of effort and dedication here, but unfortunately, I have to mark this review request as invalid.

## The issue:
**Not Professional PR as Added No custom README.**
There is no custom README added, kindly make sure that your repo looks professional, and a custom README is a must for that. Please take a look at this [template](https://github.com/microverseinc/readme-template).

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. Happy Coding 💻

File: README.md [hunk 1]
```diff
@@ -1,24 +1,75 @@
...
+### Setup
 
-* Deployment instructions
+- Install [git](https://git-scm.com/downloads)
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/EroAuditore/micro_reddit.git`
+- Install [Ruby](https://ruby-doc.org/downloads/)
+- Run `ruby console`
```
Comment (lekegitrepo) [inline README.md]: - This should be `rails console`

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @chuckbuckethead and @EroAuditore :clap: :clap: 
#### Good job :+1: The models are well implemented.
#### However, some requirements are missing/not implemented correctly. Check my comments below :point_down:

#### 1) Readme:
 - Add instruction on how to run database migration

#### _Reviewed by Temitayo Adeleke_

File: README.md (hunk 1 above)
Comment (crgc) [reply README.md]: Done!

**Comment by crgc** [comment]
Added instruction on how to run database migration, per TSE recommendation.

**Review by umairarshadbutt — COMMENTED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @chuckbuckethead and @EroAuditore ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**